services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int
+                       ) -> QuerySet:

```

Comment:
```suggestion

def create_cinema_hall(
         hall_name: str,
         hall_rows: int,
         hall_seats_in_row: int
 ) -> None:
```
This style with splitting line looks neater for me. And you do not return anything in this function